UnityProject/Assets/Scripts/Items/Others/SupermatterSliverContainer.cs

```diff
@@ -0,0 +1,84 @@
+using UnityEngine;
+
+namespace Items
+{
+	public class SupermatterSliverContainer : MonoBehaviour, ICheckedInteractable<HandActivate>
+	{
+		private const int NOT_LOADED_SPRITE = 0;
+		private const int LOADED_SPRITE = 1;
+		private const int SEALED_SPRITE = 2;
+
+		public SpriteHandler[] spriteHandlers = new SpriteHandler[] { };
+		private ItemStorage itemStorage;
+		private ItemSlot sliverSlot;
+		public bool isSealed = false;
+		public bool isLoaded = false;
+
+		private void Awake()
+		{
+			itemStorage = GetComponent<ItemStorage>();
+			sliverSlot = itemStorage.GetIndexedItemSlot(0);
+			sliverSlot.OnSlotContentsChangeServer.AddListener(OnServerSlotContentsChange);
```

Comment:
Can you add a remove in OnDisable for this listener

Reply:
Fixed